### DISCUSSION TITLE
Thermal conductivity file

### DISCUSSION THREAD

**Discussion by nourriahi** [idaholab/moose#24973] [Q&A General]
Hello Everyone,

I am trying lately to define one of my variables in the heat conduction equation which is the thermal conductivity as it looks in the graph below:

![image](https://github.com/idaholab/moose/assets/131006365/631713b6-96b3-44ad-af55-8cc3e0696359)

I have already defined this variable using a csv file (packed column) and it works, the problem for me is that I want to have a csv file which include data as in the graph, to my knowledge this is impossible as I am defining two outputs for the same input (two thermal conductivity values for one temperature), I thought  about using timeperiods to control this but I think it is not accurate as different regions will have different temperatures and the thermal conductivity values will not be the correct ones for the different regions.

Any suggestions regarding this to be able to define thermal conductivity as in the graph?

Thanks

**Comment by GiudGiud** [comment]
Hello

Do you have a variable for the phase fraction?
I think you could blend two thermal conductivities (from two CSV files for example). You just need some variable to decide the mix of each

Guillaume
